config/application.rb

```diff
@@ -16,12 +16,16 @@ class Application < Rails::Application
     # Common ones are `templates`, `generators`, or `middleware`, for example.
     config.autoload_lib(ignore: %w[assets tasks])
 
+    redis_conf = YAML.safe_load(ERB.new(File.read(Rails.root.join("config", "redis.yml"))).result, permitted_classes: [Symbol], aliases: true)["cache"]
+    redis_conf[:url] = "redis://" + redis_conf[:host] + "/" + redis_conf[:db].to_s
+    config.cache_store = :redis_cache_store, redis_conf
+
```

Comment:
Many projects also have this: `config.active_job.queue_adapter = :sidekiq`

and this:
```
    # Send all logs to stdout, which docker reads and sends to datadog.
    config.logger = Log::Logger.new($stdout) unless Rails.env.test? # we don't need a logger in test env
```